test/framework/mesh/logical_volume/lv_boolean_test1.lua

```diff
@@ -1,3 +1,10 @@
+--[[# test for boolean operations on logical volumes
+lv1 = sphere
+lv2 = right circular cylinder (rcc)
+lv3 = in rcc but not in sphere
+--]]
```

Comment:
```suggestion
--[[# 
test for boolean operations on logical volumes
lv1 = sphere
lv2 = right circular cylinder (rcc)
lv3 = in rcc but not in sphere
--]]
```
Is this a Lua block comment syntax?? Never seen it before that is interesting. I think it looks nicer to move the first line to its own line, if that doesn't break anything.